episodes/manage_images.md

```diff
@@ -30,24 +30,64 @@ Objectives
 
 ## Exercises
 
-::::{dropdown} Specifying a custom image tag.
+::::{dropdown} Software Challenges
+:open:
+
+Think about the challenges that you may have faced with using software for your work.
+
+- Is there a tool that you wanted to use that was not compatible with your operating system?
+- Have you ever struggled to install a piece of software?
```

Comment:
*<shudders as he remembers his days doing a PhD in bioinformatics before the days of Docker, conda, etc>* 😱

Reply:
I'm hoping I don't leave the class in cold sweats!